tests/appsec/test_asm_standalone.py

```diff
@@ -738,3 +737,71 @@ def test_app_dependencies_loaded(self):
         for dependency, seen in seen_loaded_dependencies.items():
             if not seen:
                 raise Exception(dependency + " not received in app-dependencies-loaded message")
+
+
+@rfc("https://docs.google.com/document/d/12NBx-nD-IoQEMiCRnJXneq4Be7cbtSc6pJLOFUWTpNE/edit")
+@features.appsec_standalone
+@scenarios.appsec_standalone
+# @flaky(context.library >= "python@2.18.0+dev", reason="APPSEC-56142")
+class Test_AppSecStandalone_UpstreamPropagation(AppSecStandalone_UpstreamPropagation_Base):
+    """APPSEC correctly propagates AppSec events in distributing tracing with DD_EXPERIMENTAL_APPSEC_STANDALONE_ENABLED=true."""
+
+    pass
```

Comment:
I believe that there is no need for this, you can keep the previous structure and have the V2 tests inherit from them, overriding just `propagated_tag` and `propagated_tag_value`, no?

Reply:
There is no need, but I prefer to keep base classes clean, with no attributes, and create final classes for each scenario, to keep simmetry